rubble/src/att/mod.rs

```diff
@@ -570,7 +926,7 @@ impl<A: AttributeProvider> AttributeServer<A> {
                                     if att.att_type == group_type && range.contains(att.handle) {
                                         cb(ByGroupAttData {
                                             handle: att.handle,
-                                            end_group_handle: att.handle, // TODO: Ask GATT where the group ends
+                                            end_group_handle: AttHandle::from_raw(0x003), // TODO: Ask GATT where the group ends
```

Comment:
Can't we just call `group_end` now?

Reply:
This is tricky due to the callbacks-calling-callbacks mess I created. I'll fix this later.